src/emulator/gui/src/settings_dialog.cpp

```diff
@@ -70,10 +94,21 @@ void draw_settings_dialog(HostState &host) {
             ImGui::SetTooltip("Uncheck the box to disable texture cache.");
         ImGui::Spacing();
         ImGui::PushItemWidth(400);
-        ImGui::InputTextWithHint("Set emulated system storage folder \n(Reboot after change for apply)", "Write your path folder here", &host.cfg.pref_path);
+        ImGui::InputTextWithHint("Set emulated system storage folder.", "Write your path folder here", &host.cfg.pref_path);
```

Comment:
Redo your changes here

Reply:
Made it a tooltip for the Apply button instead.

And, since it was bugging me, moved some of the other long lines to tooltips as well.